docker-compose.override.macos-sample.yml

```diff
@@ -6,6 +6,8 @@ services:
     image: mariadb:10.5.8
 
   wordpress:
+    ports:
+      - "8083:8083" # Storybook port number, see package.json
```

Comment:
Do we need to repeat the port exposure here? It is already exposed in the main docker.compose.yml so I would say it is redundant.